launch_ros/launch_ros/actions/node.py

```diff
@@ -178,6 +185,108 @@ def __init__(
+                continue
+            if name is not None:
+                params_without_from.append(param)
+                continue
+            raise ValueError('param Entity should have name or from attribute')
+        normalized_params.append(
+            get_nested_dictionary_from_nested_key_value_pairs(params_without_from))
+        return normalized_params
+
+    @staticmethod
+    def parse(entity: Entity, parser: Parser):
+        """Parse node."""
+        # TODO(ivanpauno): Have some common code with ExecuteProcess parsing method.
+        package = parser.parse_substitution(entity.get_attr('package'))
+        executable = parser.parse_substitution(entity.get_attr('executable'))
+        kwargs = {}
+        name = entity.get_attr('name', optional=True)
+        if name is not None:
+            kwargs['node_name'] = parser.parse_substitution(name)
+        ns = entity.get_attr('namespace', optional=True)
+        if ns is not None:
+            kwargs['node_namespace'] = parser.parse_substitution(ns)
+        prefix = entity.get_attr('launch-prefix', optional=True)
+        if prefix is not None:
+            kwargs['prefix'] = parser.parse_substitution(prefix)
+        output = entity.get_attr('output', optional=True)
+        if output is not None:
+            kwargs['output'] = output
+        args = entity.get_attr('args', optional=True)
+        if args is not None:
+            args = parser.parse_substitution(args)
+            new_args = []
+            for arg in args:
+                if isinstance(arg, TextSubstitution):
+                    text = arg.text
+                    text = shlex.split(text)
+                    text = [TextSubstitution(text=item) for item in text]
+                    new_args.extend(text)
+                else:
+                    new_args.append(arg)
+            kwargs['arguments'] = new_args
+        env = entity.get_attr('env', optional=True, types='list[Entity]')
+        if env is not None:
+            env = {e.get_attr('name'): parser.parse_substitution(e.get_attr('value')) for e in env}
+            kwargs['additional_env'] = env
+        remappings = entity.get_attr('remap', optional=True)
+        if remappings is not None:
+            kwargs['remappings'] = [
+                (
+                    parser.parse_substitution(remap.get_attr('from')),
+                    parser.parse_substitution(remap.get_attr('to'))
+                ) for remap in remappings
+            ]
+        parameters = entity.get_attr('param', types='list[Entity]', optional=True)
+        if parameters is not None:
+            kwargs['parameters'] = Node.parse_nested_parameters(parameters)
+        if_cond = entity.get_attr('if_cond', optional=True)
+        unless_cond = entity.get_attr('unless_cond', optional=True)
+        if if_cond is not None and unless_cond is not None:
+            raise RuntimeError("if and unless conditions can't be usede simultaneously")
```

Comment:
@ivanpauno consider:
```suggestion
            raise RuntimeError("if and unless conditions can't be used simultaneously")
```